contracts/FindMarketAuctionEscrow.cdc

```diff
@@ -284,13 +284,16 @@ pub contract FindMarketAuctionEscrow {
 			let ftType=saleItem.getFtType()
 			let nftInfo=saleItem.toNFTInfo()
 			let balance=saleItem.getBalance()
-			let buyer=saleItem.getBuyer()
-			let buyerName=saleItem.getBuyerName()
 			let seller=saleItem.getSeller()
 			let id=saleItem.getId()
 
-
-			emit EnglishAuction(tenant:self.getTenant().name, id: id, seller:seller, sellerName: FIND.reverseLookup(seller), amount: balance, auctionReservePrice: saleItem.auctionReservePrice,  status: status, vaultType:saleItem.vaultType.identifier, nft: nftInfo,  buyer: buyer, buyerName: buyerName, endsAt: saleItem.auctionEndsAt)
+			let buyer=saleItem.getBuyer()
+			if buyer != nil {
+				let profile = FIND.lookup(buyer!.toString())
+				emit EnglishAuction(tenant:self.getTenant().name, id: id, seller:seller, sellerName: FIND.reverseLookup(seller), amount: balance, auctionReservePrice: saleItem.auctionReservePrice,  status: status, vaultType:saleItem.vaultType.identifier, nft: nftInfo,  buyer: buyer, buyerName: profile?.getName(), buyerAvatar: profile?.getAvatar(), endsAt: saleItem.auctionEndsAt)
```

Comment:
profile.getName() is not the same name that we got before this PR. The name we get in the old version is the find name that the owner owns. This is a name the user himself can set.